pyoptsparse/pyNSGA2/pyNSGA2.py

```diff
@@ -4,9 +4,9 @@
 """
 # Compiled module
 try:
-    from . import nsga2  # isort: skip
+    from . import _nsga2  # isort: skip
```

Comment:
Is Meson changing the name of the binary? Right now the tests are failing because of the import error raised on line 36

Reply:
I have a fix coming in. It was due to my lack of knowledge about swig command line options

Reply:
Fixed